lib/wsdl.js

```diff
@@ -216,6 +216,8 @@ var ComplexTypeElement = Element.createSubClass();
 var ComplexContentElement = Element.createSubClass();
 var SimpleContentElement = Element.createSubClass();
 var SequenceElement = Element.createSubClass();
+// attribute
```

Comment:
I guess we don't need the comment here, as the name of the `var` speaks for itself 😺